main.go

```diff
@@ -42,6 +42,14 @@ type flagVal struct {
 	OverWrite_flag bool
 	Instance_flag  string
 	Verbose_flag   bool
+	File_flag      bool
+}
+
+// contains local and server paths
+type pathVal struct {
+	RootWorkingDirectoryLocal  string
+	RootWorkingDirectoryServer string
+	StartingPathServer         string
```

Comment:
I think @jlstack did that, maybe for a reason. Hopefully he might chime in here now that I've mentioned him.